contracts/eosio.system/include/eosio.system/eosio.system.hpp

```diff
@@ -948,6 +971,34 @@ namespace eosiosystem {
          [[eosio::action]]
          void setacctcpu( const name& account, const std::optional<int64_t>& cpu_weight );
 
+         /**
+          * Add names to the `account_name_blacklist` singleton.
+          *
+          * The `account_name_blacklist` singleton contains a vector of names, each of which is a pattern that
```

Comment:
By `pattern` I first thought they are something like a regular expression. They are actually sub-strings which cannot be contained in a new name. Probably it is better to say the check is sub-string check, not a pattern match check. But this is not critical.

Reply:
I tried to make it a tiny bit clearer by not using "matches": https://github.com/eosnetworkfoundation/eos-system-contracts/pull/189/commits/4032bcdd4aad7a3aba07b83792d565f4e60d7c41